treeherder/seta/management/commands/reset_seta.py

```diff
@@ -1,13 +1,18 @@
 from django.core.management.base import BaseCommand
```

Comment:
by renming this file we need to update the other file which expected to run the load_preseed command:
https://github.com/mozilla/treeherder/blob/4f53e93d8e27a0c4bff9cbd38da4138e0596cba8/treeherder/model/management/commands/load_initial_data.py#L15

In addition, how does this work for running locally?  I can see how this simplifies things in automation, possibly locally, but learning why we needed to change this and who calls this new command to solve the problem would help here.